### PR TITLE
Homepage css/html cleanup + fancier animation

### PR DESCRIPTION
**Pull request by benpixel**

### CODE REVIEW

File: server/views/index.handlebars [hunk 1]
```diff
@@ -54,11 +53,10 @@
     <div class='button-wrapper'>
 
       <div class='col-sm-4'>
-        <a href='/public/downloads/Zapsnap-1.0.1.dmg' class='button'>
+        <a href='/public/downloads/Zapsnap-1.0.0.dmg' class='button'>
```
Comment (shime) [inline server/views/index.handlebars]: please revert this back to `1.0.1`

**Comment by shime** [comment]
Thanks Benjamin! What about mobile? I saw you removed `<br/>`s I added for mobile.

File: server/views/index.handlebars (hunk 1 above)
Comment (benpixel) [reply server/views/index.handlebars]: @shime You mean 1.0.0?

Comment (shime) [reply server/views/index.handlebars]: nope, `1.0.1` is a newer version, we should link to the newest version from the download button

**Comment by benpixel** [comment]
@shime Hmm, honestly repo change/cleanup confused me (and my repo) so I had to do some weird manual merging. I actually found another way to scale the logo on mobile but I see now its still broken, will fix now.

**Comment by benpixel** [comment]
@shime check it out now

**Comment by shime** [comment]
@benjam1n thanks, `bundle.js` and `bundle.js.map` shouldn't be changed since you didn't edit anything in `client/index.js`. can you revert those changes?

File: public/main.css [hunk 2]
```diff
@@ -1,271 +1 @@
-html,
```
Comment (shime) [inline public/main.css:1]: this shouldn't be deleted, it's generated from `css/main.styl`

**Comment by shime** [comment]
Sorry for the repo restart, I didn't realize it would cause you issues.

File: public/main.css (hunk 2 above)
Comment (benpixel) [reply public/main.css:1]: Hmm, I just rewrote that, shouldn't be a problem.

```
html, body
  height: 100%
  min-height: 100%;
  margin: 0
```

Comment (shime) [reply public/main.css:1]: I thought you deleted the file, sorry. Didn't realize you minified it.

**Comment by benpixel** [comment]
@shime Ah, it's okay — I'm a GIT newbie. :) I did git reset for these two files, lemme know if its okay.

**Comment by shime** [comment]
Looks good now, thank you!

**Pull request merged**